pkgs/tools/misc/lazydocker/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "lazydocker-${version}";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "jesseduffield";
+    repo = "lazydocker";
+    rev = "v${version}";
+    sha256 = "0f062xn58dbci22pg6y4ifcdfs8whzlv2jjprxxk2ygzixrrjwnc";
+  };
+
+  modSha256 = "02n0lg28icy11a2j2rrlmp70blby0kmjas5j48jwh9h9a0yplqic";
+
+  subPackages = [ "." ];
+
+  meta = with stdenv.lib; {
+    description = "A simple terminal UI for both docker and docker-compose";
+    homepage = https://github.com/jesseduffield/lazydocker;
+    license = licenses.mit;
+    maintainers = with maintainers; [ das-g ];
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
```suggestion
```

Reply:
The [NixPkgs manual says](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes):
> If no platform is specified, the package does not have prebuilt binaries.

That doesn't sound like what we want here.

Also, the [Sample template for a new package review](https://nixos.org/nixpkgs/manual/#reviewing-contributions-sample-new-package) seems to imply that `meta.platforms` _should_ be set:

> ```
> ##### Reviewed points
> 
> - [ ] ...
> - [ ] `meta.platforms` is set
> - [ ] ...
> ```

(Interestingly, this doesn't seem to be reflected by [CONTRIBUTING.md > Submitting changes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes).)

Should this maybe be set to `platforms.all;`?

Reply:
`platforms` is already set to `platforms.all` by `buildGoModule`

https://github.com/NixOS/nixpkgs/blob/c7276b2b3d05e5191a9b5b7236d63edf49204f79/pkgs/development/go-modules/generic/default.nix#L196

Reply:
Ah, OK. :+1:

That might be worth mentioning at https://nixos.org/nixpkgs/manual/#ssec-go-modules :thinking:

Reply:
removed explicit `platforms` in 1658e84